superset/assets/cypress/integration/dashboard/dashboard_tests.js

```diff
@@ -0,0 +1,26 @@
+describe('Load dashboard', function () {
+  it('Load birth names dashboard', function () {
+    cy.server();
+    cy.login();
+
+    cy.visit('/superset/dashboard/births');
+
+    cy.route('POST', '/superset/explore_json/**').as('getJson');
+    cy.wait(7000, ['@getJson']);
+
+    let sliceData;
+
+    cy.get('@getJson.all').then((xhrs) => {
+      sliceData = xhrs;
+      xhrs.forEach((data) => {
+        expect(data.status).to.eq(200);
+        expect(data.response.body).to.have.property('error', null);
+        cy.get(`#slice-container-${data.response.body.form_data.slice_id}`);
```

Comment:
What does this line do?

Reply:
This would verify that there exists a slice container in the dashboard for each slice loaded.

Reply:
does it for sure fail in the case that it doesn't exist? is it better/safer to chain a `.then()` with an expect?

Reply:
Yes a get will fail with `CypressError: Timed out retrying: Expected to find element` the two expects above should always catch before the last one does, but seems good to have our bases covered.